styleguide/derek/_partials/solutions/announcements.ejs

```diff
@@ -0,0 +1,54 @@
+<div class="container-fluid standard-padding bg-light-gray">
+  <div class="container">
+
+    <div class="center-block">
+      <h2 class="center teal">Announcements</h2>
+    </div>
+
+    <div class="announcement-container announcement-offer row-eq-height">
+      <a href="#" class="announcement-close"></a>
```

Comment:
@runrog we need to have the "close" behavior defined. Maybe we can model this on the Bootstrap alert functionality? http://getbootstrap.com/javascript/#alerts